Formula/awscli.rb

```diff
@@ -29,8 +29,8 @@ class Awscli < Formula
   # 2. Ignore `six`. Update all other PyPI packages
 
   resource "awscrt" do
-    url "https://files.pythonhosted.org/packages/69/5f/997b6c9a8f77cf5f2f1e434a35088a4ddd119254b1a6dc8fb196e0607d66/awscrt-0.14.3.tar.gz"
-    sha256 "9cb96574ee28b7258a9fdd4beaf0a2061f5cb29945acdf2d88ad6d7de8a4e98e"
+    url "https://files.pythonhosted.org/packages/51/1f/4d9193f7b16d102c3c11d8edcb11ea741ac6562d8fcf4d1ae1aa223f197c/awscrt-0.14.0.tar.gz"
```

Comment:
https://github.com/aws/aws-cli/blob/2.7.32/setup.cfg#L39

Reply:
maybe we can just stick with the latest patch version
```
    Running setup.py install for awscrt: finished with status 'error'
  error: legacy-install-failure
```

Reply:
interesting, failed to build 0.14.3 as well

Reply:
weird that `ld` doesn't seem to print any actual warning, just:

```
ld: fatal warning(s) induced error (-fatal_warnings)
```

debugging locally, I can get it to build by dropping `-fatal_warnings`. not sure what else to try

Reply:
Weird, I actually did not have any issues of building it in my local.